.github/workflows/build.yaml

```diff
@@ -33,6 +33,12 @@ jobs:
       - name: golangci-lint
         uses: reviewdog/action-golangci-lint@v1
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        id: buildx
+        with:
+          install: true
+
```

Comment:
We also need to add this to:
https://github.com/kyverno/kyverno/blob/main/.github/workflows/image.yaml
https://github.com/kyverno/kyverno/blob/main/.github/workflows/release.yaml